### DISCUSSION TITLE
How can I load my own python package into jupyterlite? I only need to import it to use it.

### DISCUSSION THREAD

**Discussion by deepbytelab** [jupyterlite/jupyterlite#1517] [General]
After deploying jupyterlite locally, I found that I need to build my own ptyhon package, and I want to inject it into jupyterlite when I build it, so that I can use it in jupyterlite. I have tried a lot of ways, but I didn't get any solution, so I want to build it through jupyter lite, and then I can use import my_package directly in the cell.  my_package.xxx(), thanks!

**Comment by bollwyvl** [comment]
Please see https://jupyterlite.readthedocs.io/en/latest/howto/pyodide/wheels.html
